aerogear/ionic-showcase
# AEROGEAR-8695 add an application trust section to security section

### Description
# What
Create a menu entry for app trust and have it link to an app trust page and move change the security section to devicetrust (AEROGEAR-8695)'


# Why
https://issues.jboss.org/browse/AEROGEAR-8695

# Verification 
- clone the repo
- checkout the branch
- run `npm i`
- run `ionic serve`
- check the Security section of the menu drops down and you can go to the device trust and application trust pages as outlined in the screen capture below

![showcase1](https://user-images.githubusercontent.com/16667688/55950093-0143c800-5c4c-11e9-805c-247647fa6d6a.gif)





##### Checklist
